modules/postman-templating/src/parse-csv.ts

```diff
@@ -0,0 +1,55 @@
+import CSVParse from 'csv-parse'
+import { CSVParams } from './types'
+import { isEmpty } from 'lodash'
+import { 
+  RecipientColumnMissing,
+  UnexpectedDoubleQuoteError
+} from './errors'
+/**
+ * Ensures that the recipient column exists and converts headers to lowercase
+ * Deduplicates the csv by overriding the same recipient with newer records
+ * @param readStream
+ */
+export const parseCsv = async (readStream: NodeJS.ReadableStream): Promise<Array<CSVParams>> => {
```

Comment:
I'm not super familiar with Typescript but will using NodeJS types in this function have any effect in terms of being able to use this function in the browser?

Reply:
That's a good point. I don't think it can be usable in the browser. At the same time, the code for the frontend has not been written yet and i'm not sure what is the requirement for that.

Reply:
I was thinking we can change it to a different library if it doesn't work on frontend or just use `parseCsv` for backend & worker. Maybe a little short sighted, but I think we can figure out what to do at a later stage when someone is working on it.